migration/dvomigrations/mig_0005_runtimes_heartbeats_last_check.go

```diff
@@ -0,0 +1,36 @@
+
+package dvomigrations
+
+import (
+	"database/sql"
+
+	"github.com/RedHatInsights/insights-results-aggregator/migration"
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+)
+
+var mig0005CreateRuntimesHeartbeats = migration.Migration{
+	StepUp: func(tx *sql.Tx, _ types.DBDriver) error {
+		_, err := tx.Exec(`
+			CREATE TABLE runtimes.heartbeats (
```

Comment:
Not sure if we are allowed to create tables with this naming, or we need to stick to dvo ones.

Reply:
Seems that is not allowed: `schema \"runtimes\" does not exist"`

Reply:
Sticked to dvo ones for simplicity